Automattic-Tracks-iOSTests/Test Helpers/Extensions.swift

```diff
@@ -26,3 +27,11 @@ extension FileManager {
         return fileURL
     }
 }
+
+extension XCTestCase {
+    func async_test(timeout: TimeInterval, _ block: (XCTestExpectation) -> ()) {
```

Comment:
Not important:

Well done for improving this cumbersome XCTest API.

:twitch: at the underscore 😄 

You could rename this to `waitUntil` instead. Inspired by [Nimble's asynchronous expectations](https://github.com/Quick/Nimble#asynchronous-expectations).

Reply:
Ugh, totally – I've been writing too much Ruby 😅 

Renamed to `waitForExpectation` – I considered using the same api as Nimble `waitUntil(timeout: TimeInterval, done: () -> Void)`, but this felt more in line with the rest of `XCTest` IMHO, and as this is an extension vs a separate framework I felt like imitating that style made a bit more sense.

Renamed in 5da3755d8c47fe6d9f1bb12494f33552fe96ff78